recipe/0103-poetry-reading-annotations/index.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Annotating a Poetry Reading
+id: 103
+layout: recipe
+tags: [audio, presentation, annotation]
+summary: "Use annotations to indicate aspects of the performance of a particular poem."
+---
+
+
+## Use Case
+
+While an audio file of a poetry performance may be divided into a track for each poem, scholars may wish to use annotations to indicate aspects of the performance of a particular poem.  
+
+A researcher might want to annotate the following types of information:
+* structural information (introduction, title, stanzas)
+* points where the performer takes a breath
+* repeated phrases
+
+Since annotations could be available at the same time the manifest is generated, or might be a separate process that references the item manifest, both scenarios are shown.
+
+There is a third use case where manifests are unaware of annotations on them, but the systems that display the item are aware of the annotations and pull them in, using the target block in the annotation.
+
+## Implementation notes
+
+This implementation builds off of the [audio example][0002], but adds Web Annotations.
+
+This recipe shows two variations of referencing annotations from within a manifest.  The first has the annotations embedded within the manifest file.  The second has a reference to annotations in a separate file.
+
+
+## Example
+
+A manifest for the poem "Her Kind" read by Anne Sexton in 1974.  The recording is 107 minutes long.  Annotations are included in the manifest and show both a point annotation (at a particular time) and a range annotation (covering a time range).  (For more on these annotations, see the [annotation.json file description.](#annotations))
```

Comment:
```suggestion
A manifest for the poem "Her Kind" read by Anne Sexton in 1974.  The recording is 107 seconds long.  Annotations are included in the manifest and show both a point annotation (at a particular time) and a range annotation (covering a time range).  (For more on these annotations, see the [annotation.json file description.](#annotations))
```
I think you meant to put seconds here instead of minutes.